### ISSUE TITLE
COD admin boundaries in multiple files

### ISSUE THREAD

**Issue by turnerm** [mapaction/pipeline_mvp#14]
The COD admin boundaries used thus far have always been available as a single file on HDX containing multiple admin levels. However, for [Dominica](https://data.humdata.org/dataset/dominica-administrative-level-0-national-and-level-1-boundaries-form-gadm) the " dma_admbndl_ALL_py_s1_dominode.zip" file only contains lines, while the polygons are split into separate admin 0 and 1 level files. 

The pipeline needs to be modified to handle this possibility, while still keeping a single file as the default since it's more common.

**Comment by turnerm** [comment]
This issue is also present with [Vanuatu](https://data.humdata.org/dataset/vanuatu-administrative-level-0-1-and-2-boundaries) and [Fiji](https://data.humdata.org/dataset/fiji-administrative-boundaries), although in both these cases a geodatabase with all admin levels is available

**Comment by janvdvegt** [comment]
Yeah, to incorporate this we need to be able to pass folders as well, instead of just files. I started working on this but will not be able to finish this, I will add it to the document as well

**Comment by turnerm** [comment]
@janvdvegt just to be clear, this is a separate issue from the one affecting the output files (where only .shp files are moved to the finally directory). This issue is only present for a handful of countries and concerns the data extraction; while the other issue affects all countries and impacts the final product.
